engine/node_matlab.cc

```diff
@@ -493,6 +493,20 @@ namespace MathDAG
     checkArg(0,0);
     return o<<"frac("<<arguments[0][0]->matlab()<<")";
   }
+  
+  template <>
+  ostream& OperationDAG<OperationType::percent>::matlab(ostream& o) const
+  {
+    checkArg(0,0);
+    return o<<"("<<arguments[0][0]->matlab()<<")%";
```

Comment:
I think in matlab, you'd need to write this as 0.01*(arguments[0][0]->matlab())

Reply:
Need to fix this to be 100* times now. Will do so just now.